tests/lib/ocpp/v201/json/grid/README.md

```diff
@@ -0,0 +1,137 @@
+# Grid Profile 
+
+This is a foundational Profile designed to act as a sort of temporal grid paper for testing out different
```

Comment:
This is what I was getting at in my earlier comment about READMEs. It would be helpful to have something like this in each of these directories.

Reply:
Dig it.